docs/menu.yml

```diff
@@ -252,6 +252,8 @@ catalog:
                 path: "/en/setup/backend/backend-mongodb-monitoring"
               - name: "BookKeeper"
                 path: "/en/setup/backend/backend-bookkeeper-monitoring"
+              - name: "RocketMQ"
+                path: "/en/setup/backend/backend-rocketmq-monitoring"
```

Comment:
MQ should be in `MQ monitoring`, you are putting this in the Database. Please fix.